core-bundle/tests/Controller/AbstractBackendControllerTest.php

```diff
@@ -262,8 +262,8 @@ function (string $template, array $context) use ($expectedContext) {
                     }
 
                     $map = [
-                        '@ContaoCore/Backend/be_menu.html.twig' => '<menu>',
-                        '@ContaoCore/Backend/be_header_menu.html.twig' => '<header_menu>',
+                        '@Contao/backend/chrome/main_menu.html.twig' => '<menu>',
+                        '@Contao/backend/chrome/header_menu.html.twig' => '<header_menu>',
```

Comment:
Why is the namespace for the menus called `chrome` and not `menu`? Just curious 😄

Reply:
I would like to put anything there, that is from the "backend ui", i.e. the things around the main section. Currently only the menus have individual templates, but this could also include popup messages, the header bar etc.